package.json

```diff
@@ -56,7 +56,7 @@
     "prop-types": "^15.5.10",
     "raven-js": "^3.19.1",
     "rc-slider": "^8.3.1",
-    "react": "^15.6.1",
+    "react": "^16.1.0",
```

Comment:
react charts https://github.com/RakanNimer/react-google-charts/issues/161 does not work without